samples/attested_tls/common/verify_callback.cpp

```diff
@@ -266,15 +253,15 @@ int verify_callback(int preverify_ok, X509_STORE_CTX* ctx)
         (int)(buff - der),
         der_len);
 
-#if 1 // for debugging purpose
+#if 0 // for debugging purpose
```

Comment:
should not have 'if 0' code.  Just delete the code if you don't want it.
If you want the developer to be able to enable it use a specific #define and not if 1 or 0